package.json

```diff
@@ -55,6 +55,9 @@
     "e2e": "jest --config jest/e2e.js",
     "e2e-ci": "jest --config jest/e2e-ci.js --ci -w 1",
     "tsc": "tsc",
+    "nyc": "./node_modules/.bin/nyc",
+    "codecov": "./node_modules/.bin/codecov",
+    "posttest": "yarn run nyc report --report=json && yarn run codecov -f coverage/*.json",
```

Comment:
Don't call it `posttest` - `post` and `pre` have special meaning in a package.json - basically when you run `yarn foo`, first it checks to see if `prefoo` exists - if it does, it runs it. Then it runs `foo`, and finally `postfoo` if it exists.